### ISSUE TITLE
Question about building typ probably missleading

### ISSUE THREAD

**Issue by hmartink** [westnordost/StreetComplete#1349]
Often the first question about a building is "What was this building built for". Often I found I cannot answer this question truly, which is sad, as I can easily answer the follow up questions (number of floors and roof type). A better question might be "What is this building used for".

Long story:
1) There are lots of older buildings around. I have *no* clue what they once were built for. I know one building, which looks like an ordinary multiparty living home, but I know it used to be a store. It was remodeled a few years ago, the front glass replaced by walls and insulation etc. So if I would answer this question now I would (wrongly) answer "multiparty living home" instead of "store" (sorry, I don't know the exact osm terms).

2) Why is the original build type so important? We have buildings which used to be farms (maybe 50 years ago or more, I don't know) but now are used for flats. No animals, no fields, no farmer. Would'nt it be better to ask what the building is used for now?

3) This question is difficult. I might need to walk around the house, on their premise, to really find out. I suggest to start with more easy questions which I can answer from the street side easily, i.e. number of floors and roof type.

(If this question was wrongly translated from English into German, then 1) and 2) can probably be fixed with an translation update).

To reproduce go to an unmapped house and tip on it. You get the question: "Als was wurde dieses Gebäude errichtet" (What was this building errected for)

I use version 10.1 of StreetComplete

**Comment by matkoniecz** [comment]
> There are lots of older buildings around. I have no clue what they once were built for.

I know the pain and I disabled this quest for myself.

> Why is the original build type so important?

That is how `building` tag is defined and this quest is adding value for that tag.

> Would'nt it be better to ask what the building is used for now?

It may be a good idea to collect and use in queries current purpose (`building:use`) as it is easier to collect, more useful and important, with fallback to `building` in queries.

> What is this building used for

That would answer different question and would be added as `building:use` tag.

> I suggest to start with more easy questions

This would change situation only for you - but are you aware that you can change quest order in settings?

**Comment by westnordost** [comment]
@hmartink: The question is asked this way because the `building` tag is defined this way. 
That being said, I find it is no problem at all to tag it according to the current usage if it is not obvious that it ever was built for a different purpose. Surveyors are not clairvoyant and it can always be corrected later if someone finds out later that it was built for a different purpose once. For more than 99% of buildings, the current usage is equal to the original purpose anyway. 
Recently, I found this [Denkmalliste of the city of Hamburg](https://www.hamburg.de/contentblob/3947920/d64e9b8843d6e9bf27895320abd1f00b/data/denkmalliste-gesamt.pdf) which also included as what any building has been constructed. On going through this list, I found a few buildings that had a different purpose then than now, but it was no problem at all to correct this in a blink.

So, long story short, perhaps the question could be reworded to less explicitly ask for what the building was constructed as, but still be clear about that in the rare case the current and intended usage is not the same, the user should select the intended usage (or better even, leave a note, so that both can be recorded in the OSM database later). Any ideas?

**Comment by westnordost** [comment]
How about simply "What kind of building is this?"

In cases where the surveyor sees a church that has been converted to an office, even with that wording, I don't think he would answer it rather with "office" than with church. In the best case, the surveyor is insecure and leaves a note, which is the best option in a case like this, so that both building and building:use can be recorded.
